lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/custom_element_embedding_strategy.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:ui/src/engine/dom.dart';
 
+import '../hot_restart_cache_handler.dart';
```

Comment:
```suggestion
import '../hot_restart_cache_handler.dart' show registerElementForCleanup;
```

That way it's more clear (to me) that you're importing that file specifically for the function you want from it!

Reply:
Done